platform/tests/Platform.ApiTests/Steps/EstablishmentTrustsSteps.cs

```diff
@@ -35,120 +32,139 @@ private void GivenAValidTrustRequestWithId(string id)
-            Method = HttpMethod.Post
-        });
+        await _api.Send();
     }
 
-    [Then("the trusts search result should be ok")]
-    private void ThenTheTrustsSearchResultShouldBeOk()
+    [Then("the trust result should be ok and have the following values:")]
+    private async Task ThenTheTrustResultShouldHaveValues(Table table)
     {
-        var result = _api[SearchRequestKey].Response;
+        var response = _api[RequestKey].Response;
 
-        result.Should().NotBeNull();
-        result.StatusCode.Should().Be(HttpStatusCode.OK);
+        response.Should().NotBeNull();
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+
+        var content = await response.Content.ReadAsByteArrayAsync();
+        var result = content.FromJson<Trust>();
+
+        table.CompareToInstance(result);
     }
 
-    [Given("an invalid trusts suggest request")]
-    private void GivenAnInvalidTrustsSuggestRequest()
+    [Then("the trust result should be not found")]
+    private void ThenTheTrustResultShouldBeNotFound()
     {
-        var content = new { Size = 0 };
+        var result = _api[RequestKey].Response;
 
-        _api.CreateRequest(SuggestInvalidRequestKey, new HttpRequestMessage
-        {
-            RequestUri = new Uri("/api/trusts/suggest", UriKind.Relative),
-            Method = HttpMethod.Post,
-            Content = new StringContent(content.ToJson(), Encoding.UTF8, "application/json")
-        });
+        result.Should().NotBeNull();
+        result.StatusCode.Should().Be(HttpStatusCode.NotFound);
     }
 
-    [Then("the trusts suggest result should have the follow validation errors:")]
-    private async Task ThenTheTrustsSuggestResultShouldHaveTheFollowValidationErrors(Table table)
+    [Then("the trust suggest result should be ok and have the following values:")]
+    private async Task ThenTheTrustsSuggestResultShouldShouldHaveValues(Table table)
     {
-        var response = _api[SuggestInvalidRequestKey].Response;
+        var response = _api[SuggestRequestKey].Response;
 
-        response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
+        response.Should().NotBeNull();
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
 
         var content = await response.Content.ReadAsByteArrayAsync();
-        var results = content.FromJson<ValidationError[]>();
-        var set = new List<dynamic>();
+        var results = content.FromJson<SuggestResponse<Trust>>().Results;
+        var result = results.FirstOrDefault();
+        result.Should().NotBeNull();
 
-        foreach (var result in results)
+        var actual = new
         {
-            set.Add(new { result.PropertyName, result.ErrorMessage });
-        }
+            result?.Text,
+            result?.Document?.TrustName,
+            result?.Document?.CompanyNumber
+        };
 
-        table.CompareToDynamicSet(set, false);
+        table.CompareToInstance(actual);
```

Comment:
Is it possible to use `result` instead of `actual` here and below, or are there too many fields that are out of our control in the response?